lib/node_modules/@stdlib/math/base/special/cexp/README.md

```diff
@@ -99,16 +101,14 @@ var re;
 var im;
 var z1;
 var z2;
-var o;
 var i;
 
 for ( i = 0; i < 100; i++ ) {
     re = round( randu()*100.0 ) - 50.0;
```

Comment:
Instead of `round( randu() )`, would you mind updating this example to use `discrete-uniform`?